.gitignore

```diff
@@ -16,6 +16,7 @@ target/
 /native/test/test
 /native/*.o
 /native/*.so
+*.xml
```

Comment:
maybe just ignore `cobertura.xml` so future xml files are not black listed?

Reply:
Sure